cmake/OpenCVInstallLayout.cmake

```diff
@@ -23,28 +23,28 @@ if(ANDROID)
 elseif(WIN32 AND CMAKE_HOST_SYSTEM_NAME MATCHES Windows)
 
   if(DEFINED OpenCV_RUNTIME AND DEFINED OpenCV_ARCH)
-    set(_prefix "${OpenCV_ARCH}/${OpenCV_RUNTIME}/")
+    ocv_update(OpenCV_INSTALL_BINARIES_PREFIX "${OpenCV_ARCH}/${OpenCV_RUNTIME}/")
```

Comment:
Lets use uppercase variant for all new CMake configurable variables: `OPENCV_INSTALL_BINARIES_PREFIX`